modules/library/coverage/src/main/java/org/geotools/coverage/processing/OperationJAI.java

```diff
@@ -1079,4 +1099,86 @@ final RenderedImage getSource() {
             return null;
         }
     }
+    
+    protected void handleJAIEXTParams(ParameterBlockJAI parameters, ParameterValueGroup parameters2) {
+        return;
+    }
+    
+    protected static Map<String, Object> handleROINoDataProperties(Map<String, Object> properties,
```

Comment:
Protected methods are API, please add javadoc (also for the method below this one)